### ISSUE TITLE
Request one child document at a time [phab:D1520]

### ISSUE THREAD

**Issue by kythe** [google/kythe#1934]
_Created by **zarko** at 2017-04-13 21:53:37_: 

When include_children is used with Documentation,
nodes with moderate numbers of children fail because
they quickly hit the default 20 ticket limit for
internal calls. This diff changes slowMultilevelDocumentation
to issue one Documentation call per child to try and
stay below the limit.

**Comment by kythe** [comment]
_**zarko** wrote at 2017-04-13 22:06:36_:

  - Add a TODO.

**Comment by kythe** [comment]
_**fromberger** commented inline (context lost during migration) at 2017-04-13 22:59:03_:

```
func mergeDocumentationReply(from, into *xpb.DocumentationReply) {
```

**Comment by kythe** [comment]
_**fromberger** accepted this revision at 2017-04-13 22:59:03_

**Comment by kythe** [comment]
_**zarko** wrote at 2017-04-13 23:08:43_:

  - Address comment.

**Comment by kythe** [comment]
_**zarko** closed via commit af4d74dacaa96650b5a67408cb9e4100f7e5081b at 2017-04-17 12:08:19_
